app/models/user.rb

```diff
@@ -336,6 +349,31 @@ def flipper_id
 
   private
 
+  def mpi_profile
+    if mpi.nil?
+      Rails.logger.info "[User] Cannot find MPI record for User with uuid: #{uuid}"
```

Comment:
this is going to be super noisy.  MPI will be nil for all users that are not LOA3 and this will be called a LOT.

Reply:
Ah yeah I'll remove this log. I was wondering in what cases MPI was nil